src/ui_utils.c

```diff
@@ -2097,7 +2097,9 @@ static void on_config_file_clicked(GtkWidget *widget, gpointer user_data)
 		if (g_file_test(global_file, G_FILE_TEST_EXISTS))
 			g_file_get_contents(global_file, &global_content, NULL, NULL);
 
-		document_new_file(utf8_filename, ft, global_content);
+		// open or create the document and mark it as changed if it didn't already exist
+		GeanyDocument *doc = document_new_file(utf8_filename, ft, global_content);
+		document_set_text_changed(doc, ! g_file_test(file_name, G_FILE_TEST_EXISTS));
```

Comment:
Ah my bad, I was trying to make it open a new blank CSS file with the correct filename. I agree we shouldn't pre-populate with everything, it'd be better to just put a comment in them like:

> Customize this file to override the system default file.
> See `$prefix/share/geany/whatever.file` for examples of what can go here